pages.it/common/git-rebase.md

```diff
@@ -0,0 +1,37 @@
+# git rebase
+
+> Applica i commit di un ramo su un ramo differente.
+> Tipicamente usato per riallineare (rebase) due rami, creando copie dei commit nella nuova posizione.
+> Maggiori informazioni: <https://git-scm.com/docs/git-rebase>.
+
+- Riallinea il ramo corrente con il ramo principale:
+
+`git rebase {{master}}`
```

Comment:
This page is being edited here: #4533 Maybe you want to delay your merge.

Reply:
Sure, it can wait :)